examples/tutorial/app/shared_context.py

```diff
@@ -54,9 +56,11 @@ def __init__(self):
         self.show()
 
     def on_init(self, event):
-        self.text = Text('Initialized', font_size=40.,
-                         anchor_x='left', anchor_y='top',
-                         parent=[self.vb_0.scene, self.vb_1.scene])
+        self.text = Text('Initialized', font_size=10.,
+                         anchor_x='left', anchor_y='top')
+        self.views = [self.text.view(), self.text.view()]
+        self.vb_0.add(self.views[0])
+        self.vb_1.add(self.views[1])
```

Comment:
@campagnola this does not work, and I tried a few things. What is the proper syntax? There are also rendering issues even with a single `vb_0.add(self.text)`, but that might be harder to fix.